packages/dashboard/src/customization/widgets/lineSymbol/component.test.tsx

```diff
@@ -0,0 +1,221 @@
+import React from 'react';
+import { fireEvent, render, waitFor } from '@testing-library/react';
+import { DndProvider, useDrag } from 'react-dnd';
+import { HTML5Backend } from 'react-dnd-html5-backend';
+import LineWidgetComponent from './component';
+import { Provider } from 'react-redux';
+import { DashboardStore, configureDashboardStore } from '~/store';
+import { DashboardState } from '~/store/state';
+import { LineProperties, LineWidget } from '../types';
+import { act } from 'react-dom/test-utils';
+
+const mockWidgetPartial = {
+  id: 'mock-line-widget',
+  type: 'line',
```

Comment:
I think this is the line chart type? Do the types clash?

Reply:
the line chart has `type: 'line-chart'`